### ISSUE TITLE
Accessibility: Add an option to display the on/off state of toggle buttons in the Action Center.

### ISSUE THREAD

**Issue by DrewNaylor** [DrewNaylor/Retiled#123]
The text will be to the right of the icon in left-to-right languages as if it were like icons in lists (#110), with the reverse in RTL and on the right side. There will have to be a way to specify that these two settings are different. The other one can be "Show icons beside text when available", and this one can be "Show Action Center Quick Action state as text beside each icon when available".

So basically, it'll be:
```text
🔦 off

FLASHLIGHT
```

And:
```text
🔦 on

FLASHLIGHT
```

Instead of:
```text
🔦

FLASHLIGHT
```

But imagine the accent color also on the ones that are on.

This will help people using screen readers and/or people who are colorblind and/or people that need extra labels on things to help them figure out what's going on faster.
